protocol/src/main/java/org/geysermc/mcprotocollib/protocol/ServerListener.java

```diff
@@ -183,6 +182,7 @@ public void packetReceived(Session session, Packet packet) {
                 protocol.setState(ProtocolState.CONFIGURATION);
             } else if (packet instanceof ServerboundPingRequestPacket pingRequestPacket) {
                 session.send(new ClientboundPongResponsePacket(pingRequestPacket.getPingTime()));
+                session.disconnect(Component.translatable("multiplayer.status.request_handled"));
```

Comment:
What's the reasoning here?

Reply:
![image](https://github.com/GeyserMC/MCProtocolLib/assets/40795980/3b43901b-2bdc-4867-8175-aed1216c1fec)
That's just what vanilla does

Reply:
nvm wrong direction, lemme get you the right one

Reply:
![image](https://github.com/GeyserMC/MCProtocolLib/assets/40795980/a42e651a-1d95-4005-848c-8e2e82044b63)
![image](https://github.com/GeyserMC/MCProtocolLib/assets/40795980/f8d49b39-63f0-4f5c-be26-c439cd11e8a0)

Reply:
fair enough